.travis.yml

```diff
@@ -1,39 +1,88 @@
 language: php
+sudo: false
 php:
-- 5.3
-- 5.4
-- 5.5
-- 5.6
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
+  - 7.0
+
 env:
-  matrix:
+  # trunk:
   - WP_VERSION=latest WP_MULTISITE=0
   - WP_VERSION=latest WP_MULTISITE=1
-  - WP_VERSION=4.1 WP_MULTISITE=0
-  - WP_VERSION=4.1 WP_MULTISITE=1
-  - WP_VERSION=4.0 WP_MULTISITE=0
-  - WP_VERSION=4.0 WP_MULTISITE=1
+  # latest stable:
+  - WP_VERSION=4.3 WP_MULTISITE=0
+  - WP_VERSION=4.3 WP_MULTISITE=1
+  # previous stable:
+  - WP_VERSION=4.2 WP_MULTISITE=0
+  - WP_VERSION=4.2 WP_MULTISITE=1
+  # earliest supported:
   - WP_VERSION=3.9 WP_MULTISITE=0
   - WP_VERSION=3.9 WP_MULTISITE=1
-  global:
-  - PHPCS_DIR=/tmp/phpcs
-  - secure: SMPml9xPNS2AGYZyRzGU83qdH85tBI4VB4mTtRISE1DB64Tjg9WcD1Zrztw2BlynARgHNZD/BzVDVhQvIjzhRWvqPGTGw22RgikaZJV+eCBMaZ4VIP9uSdsgCWRpFkPdJbnVARfS9eWEehaacVOVM0CKRN8G3GQYWLVE5+m1LIw=
-sudo: false
+
+matrix:
+  allow_failures:
+    - php: hhvm
+    - php: 7.0
+  fast_finish: true
+  exclude:
+    # don't test 3.9 on hiphop:
+    - php: hhvm
+      env: WP_VERSION=3.9 WP_MULTISITE=0
+    - php: hhvm
+      env: WP_VERSION=3.9 WP_MULTISITE=1
+
+cache:
+  directories:
+  - vendor
+  - $HOME/.composer/cache
+
+before_install:
+  # set up WP install
+  - export WP_DEVELOP_DIR=/tmp/wordpress/
+  - mkdir -p $WP_DEVELOP_DIR
+  - git clone --depth=1 git://develop.git.wordpress.org/ $WP_DEVELOP_DIR
+  # set up tests config
+  - cd $WP_DEVELOP_DIR
```

Comment:
Don't we also need to checkout the correct version tag here so that we're testing against the version of WordPress in `WP_VERSION`? In the current config I don't see how Travis would know to get the version of WordPress we're supposed to be testing against?